.github/workflows/cd.yml

```
@@ -70,7 +75,7 @@ jobs:
             set -e
             
             docker pull ${{ secrets.DOCKER_HUB_USERNAME }}/ufo_fi_be
-                        
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)